src/models/events.ts

```diff
@@ -354,10 +354,10 @@ type When = Time | Timespan | Date | Datespan;
  * Type representing the different objects representing time and duration when creating events.
  */
 type CreateWhen =
-  | Omit<Time, 'type'>
-  | Omit<Timespan, 'type'>
-  | Omit<Date, 'type'>
-  | Omit<Datespan, 'type'>;
```

Comment:
Is this change expected?

Reply:
Yes this is related to #551, I think the tests never ran on that PR so this was uncaught.